ci/appveyor-install.bat

```diff
@@ -17,6 +17,7 @@ libgtk3-devel,^
 ninja,^
 python3-pip,^
 vala,^
+cmake,^
```

Comment:
These lines are alphabetically sorted.  Please add cmake in the appropriate place.

Reply:
should be fixed with the last commit